metadata_service/api/swagger_doc/template.yml

```diff
@@ -308,6 +312,10 @@ components:
             description: 'Badges associated with the Dashboard'
             items:
               $ref: '#/components/schemas/TagFields'
+        recent_view_count:
+          type: int
+          description: 'Dashboard recent view count'
```

Comment:
the recent_view_count name is a bit confusing? is it for last two weeks?

Reply:
> the recent_view_count name is a bit confusing? is it for last two weeks?

Yeah, it is actually intentional as it's hard to make this value precise for it is relative value. In other words, actual `recent` value (either 2 weeks or 4 weeks) depends on how accurately `databuilder` ingests and remove stale data. To clarify why I used `recent` here is to show this is not a total view count. WDYT?

Reply:
yeah, I understand that part, try to see if we could have more precise term. Maybe you could add a comment to make that explict that it depends on databuilder ingestion?

Reply:
Sure, will do!